python/ccxt/bybit.py

```diff
@@ -5229,33 +5246,49 @@ def set_margin_mode(self, marginMode, symbol: Optional[str] = None, params={}):
     def change_margin_type(self, symbol, is_cross, leverage, is_long):
         self.load_markets()
         symbol = self.find_symbol(symbol)
-        return self.classify_change_margin(symbol, is_long, is_cross, leverage)
+        enableUnifiedMargin, enableUnifiedAccount = self.is_unified_enabled()
+        if enableUnifiedAccount and not self.is_inverse():
+            # TODO: get from account and avoid _change_margin_type/force_change_margin
+            if is_cross:
+                self.set_leverage(symbol, leverage=leverage)
+                return self._change_margin_type(is_cross)
+            else:
+                return self.classify_change_margin(symbol, is_long, is_cross, leverage, force_change_margin=True)
```

Comment:
instead of calling with force_change_margin, keep the original classify_change_margin as is and simply call 
```
result = self._change_margin_type(is_cross, symbol=symbol)
return self.set_leverage(symbol, long_leverage=leverage, short_leverage=leverage)
```

Reply:
Can't do it, it will change both short and long which is really bad.